model/task/task.go

```diff
@@ -769,28 +769,28 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 }
 
-func displayTaskPriority(status string) int {
-	switch status {
-	case evergreen.TaskStarted:
-		return 10
-	case evergreen.TaskUndispatched:
-		return 40
+func displayTaskPriority(t Task) int {
```

Comment:
make this a method on task.

Reply:
done.

Reply:
I disagree. This makes us keep around potentially large slices of task documents when we only need the status

Reply:
But if we're sorting tasks there must be a slice of tasks somewhere.